Lib/test/test_socket.py

```diff
@@ -5223,7 +5223,17 @@ def checkNonblock(self, s, nonblock=True, timeout=0.0):
             self.assertTrue(
                 fcntl.fcntl(s, fcntl.F_GETFL, os.O_NONBLOCK) & os.O_NONBLOCK)
             if timeout == 0:
+                # timeout == 0: means that getblocking() must be False.
                 self.assertFalse(s.getblocking())
+            else:
+                # If timeout > 0, the socket will be in a "blocking" mode
+                # from the standpoint of the Python API.  For Python socket
+                # object, "blocking" means that operations like 'sock.recv()'
+                # will block.  Internally, file descriptors for
```

Comment:
I don't think there's any point in explaining this inside `test_socket`.

Reply:
In this test it makes sense, because I use fcntl to check the blocking mode of the underlying FD.  Usually comments don't hurt in tests.

Reply:
Pushed an update.